utilities/backupable/backupable_db.cc

```diff
@@ -1950,12 +1967,12 @@ Status BackupEngineImpl::CopyOrCreateFile(
   uint64_t processed_buffer_size = 0;
   do {
     if (stop_backup_.load(std::memory_order_acquire)) {
-      return Status::Incomplete("Backup stopped");
+      return status_to_io_status(Status::Incomplete("Backup stopped"));
```

Comment:
Why not just IOStatus::Incomplete?

Reply:
IOStatus does not have the function like Abort and Incomplete. They are not the typical IO related Status.